### PR TITLE
Bump action version to 3.3.2

### PR DESCRIPTION
**Pull request by bethanyj28**
## Description

Bumping the actions/cache version to 3.3.2 to release https://github.com/actions/cache/pull/1217

## Motivation and Context

Fixes https://github.com/actions/cache/issues/810

## How Has This Been Tested?

Testing details in https://github.com/actions/toolkit/pull/1484

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (add or update README or docs)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by yacaovsnc — APPROVED** [review]

**Review by pfleidi — APPROVED** [review]

**Comment by Ashrafmohdiit** [comment]
Hi

File: RELEASES.md [hunk 1]
```diff
@@ -107,3 +107,7 @@
 ### 3.3.1
 
 - Reduced segment size to 128MB and segment timeout to 10 minutes to fail fast in case the cache download is stuck.
+
+### 3.3.2
```
Comment (EllaMartinez1981) [inline RELEASES.md:111]: @EllaMartinez1981

File: RELEASES.md [hunk 2]
```diff
@@ -107,3 +107,7 @@
...
+
+- Fixes bug with Azure SDK causing blob downloads to get stuck.
```
Comment (EllaMartinez1981) [inline RELEASES.md:113]: iella.1.-1.PDF

Ella_Intelligent_Global_Network_Inc_230714_034803_230913_050808.pptx

en-US-default.help.txt

ThirdPartyNotices.txt

Ella Scan Sep 19, 2023.pdf

**Review by EllaMartinez1981 — APPROVED** [review]
@EllaMartinez1981

**Pull request merged**